### DISCUSSION TITLE
basic-ftp issue

### DISCUSSION THREAD

**Discussion by p3im4n** [npm/feedback#292] [Website] [answered]
hi
I used basic-ftp
 my web server
I install package with ansible
and got some error
these are the pictures of my situation:

![playbook](https://user-images.githubusercontent.com/49278298/111805288-1ddfe480-88e6-11eb-93ed-7e312df564ce.PNG)

![npm list](https://user-images.githubusercontent.com/49278298/111805356-2c2e0080-88e6-11eb-9749-505f2ae6f4e6.PNG)

![chrome](https://user-images.githubusercontent.com/49278298/111805702-8202a880-88e6-11eb-9e38-7854ebafc064.PNG)

![IE](https://user-images.githubusercontent.com/49278298/111805714-86c75c80-88e6-11eb-8e3d-a1f442fcedb2.PNG)

is any body have an idea what the problem is?

**Comment by MylesBorins** [comment, marked as answer]
@p3im4n this discussions for is for making product suggestions to the npm registrty, website, and CLI, not for general technical support.

If you are looking for help with a specific package you can try the repo for the package itself `npm repo basic-ftp` or you can try asking on https://github.community or stackoverflow.

Thanks for stopping by

**Reply by MylesBorins** [reply]
as an aside both `fs` and `require` are Node.js built-ins and you do not want to be installing them from npm
